src/unistd.rs

```diff
@@ -578,6 +562,55 @@ pub fn chown<P: ?Sized + NixPath>(path: &P, owner: Option<Uid>, group: Option<Gi
     Errno::result(res).map(drop)
 }
 
+/// Change the ownership of the file at `path` to be owned by the specified
+/// `owner` (user) and `group` (see
+/// [lchown(2)](http://pubs.opengroup.org/onlinepubs/9699919799/functions/lchown.html)).
+/// If `path` names a symbolic link, then ownership of the symbolic link file itself is changed.
+///
+/// The owner/group for the provided path name will not be modified if `None` is
+/// provided for that argument.  Ownership change will be attempted for the path
+/// only if `Some` owner/group is provided.
+#[inline]
+pub fn lchown<P: ?Sized + NixPath>(path: &P, owner: Option<Uid>, group: Option<Gid>) -> Result<()> {
```

Comment:
`lchown` is redundant with `fchownat`, which is being added as part of PR #955 .  Unless there's a reason why `fchownat` won't work for you, you shouldn't bother with `lchown`.

Reply:
OK